joomla/joomla-cms
# fixes overridebility of com_media

Fixes administrator/components/com_media/views/images/tmpl/default.php override failing
#### Test instructions:

Apply this patch with patch tester
Go to protostar/html and create a folder com_media
Create another folder named images
Copy the content of the file administrator/components/com_media/views/images/tmpl/default.php to the newly created file
Edit the file so the first line above the first form tag to be `<div><h1>This is a Protostar Override</h1></div>`
Now try to login in the fronted and edit an article.
Go to Images and link and try to select an image
If you see the title This is a Protostar Override in the modal the you have a successful test!
